.agent/specs/002219.md

```
@@ -0,0 +1,647 @@
+  {"type": "exemplar_conv_id", "value": "conv-uuid-2"},
+  {"type": "exemplar_conv_id", "value": "conv-uuid-3"},
+  {"type": "member_conv_id", "value": "conv-uuid-4"},
+  {"type": "member_conv_id", "value": "conv-uuid-5"}
+]
+```
+
+각 entry 필드:
+
+| 필드 | 타입 | 의미 |
+|---|---|---|
+| `type` | `"keyword" | "exemplar_conv_id" | "member_conv_id"` | source kind |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)